client/ayon_core/hosts/nuke/plugins/publish/increment_script_version.py

```diff
@@ -13,6 +13,8 @@ class IncrementScriptVersion(pyblish.api.ContextPlugin):
     hosts = ['nuke']
 
     def process(self, context):
+        if context.data.get("headless_farm", False):
```

Comment:
Should this really be skipped?

Reply:
Yeah, the workflow is to have a timestamped workfile and continue from the currently versioned workfile.